auacm/app/static/js/controllers/EditProblemController.js

```diff
@@ -65,6 +75,8 @@ app.controller('EditProblemController', ['$scope', '$route', '$http',
             fd.append('sol_file', $scope.solFile);
         if (typeof prob.appeared !== 'undefined')
             fd.append('appeared_in', prob.appeared);
+        if (typeof prob.comp_release !== 'undefined');
```

Comment:
You're right. I hate to be that guy, but a frontend test might have caught this...